russian/girlfriend-pinch-you/call_girlfriend/call.md

```diff
@@ -1 +1,3 @@
 Вы зовете вашу девушку, но никто не отвечает. Внезапно из сада раздается сдавленный крик ...
+
+[Пойти в сад] (go_to_the_garden/action.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.